src/app/pages/loading-waiting-states/skeleton-loader/skeleton-loader.component.html

```diff
@@ -114,7 +111,7 @@ <h2 class="example-title"></h2>
                             >check_circle
                         </mat-icon>
                         <div class="time-section" pxb-left-content>
-                            <div>8:05 <span class="ampm">AM</span></div>
+                            <div>8:05<span class="ampm">AM</span></div>
```

Comment:
I would suggest avoiding using the classes from the skeleton library for styling our main content.

Reply:
"ampm" is not part of the placeholder skeleton library...